featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -142,8 +142,7 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
     # handle loading entityset
     from featuretools.entityset.entityset import EntitySet
     if not isinstance(entityset, EntitySet):
-        if entities is not None and relationships is not None:
```

Comment:
Just wondering--why not keep the check for whether entities is not None here? Is there a world in which having no entities wouldn't end up erroring? 

If not, it might be nice to catch this with its own error instead of `Entity transactions does not exist in entityset`, which is where it gets caught in the test below.

Reply:
Sure, if we catch there we can give a more descriptive error